components/listings/image-carousel.tsx

```
@@ -0,0 +1,156 @@
+      <div className="aspect-[4/3] relative bg-muted/20">
+        {showVideo ? (
+          <div className="w-full h-full relative">
+            <video
+              ref={videoRef}
+              src={video}
+              className="w-full h-full object-contain"
+              loop
+              playsInline
+              muted={isMuted}
+              onClick={togglePlayPause}
+            />
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)